src/page.js

```diff
@@ -0,0 +1,135 @@
+// eslint-disable-next-line
```

Comment:
- [ ] Please avoid disabling Linters. There are great tools that flag programming bugs. Rather, fix them.